packages/framework/build.xml

```diff
@@ -748,34 +786,39 @@
     </target>
 
     <target name="product-search-create-structure" description="Creates structure for searching via elasticsearch." hidden="true">
-        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
-            <arg value="${path.bin-console}"/>
-            <arg value="shopsys:product-search:create-structure"/>
-        </exec>
+        <echo level="warning" message="This phing target is deprecated since Shopsys Framework 9.0.0, use 'elasticsearch-index-create' instead."/>
+        <phingcall target="elasticsearch-index-create">
+            <property name="elasticsearch.index" value="product"/>
+        </phingcall>
     </target>
 
     <target name="product-search-delete-structure" depends="production-protection" description="Deletes structure for searching via elasticsearch." hidden="true">
-        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
-            <arg value="${path.bin-console}"/>
-            <arg value="shopsys:product-search:delete-structure"/>
-        </exec>
+        <echo level="warning" message="This phing target is deprecated since Shopsys Framework 9.0.0, use 'elasticsearch-index-delete' instead."/>
+        <phingcall target="elasticsearch-index-delete">
+            <property name="elasticsearch.index" value="product"/>
+        </phingcall>
     </target>
 
-    <target name="product-search-export-products" description="Exports all products for searching via elasticsearch.">
-        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
-            <arg value="${path.bin-console}"/>
-            <arg value="shopsys:product-search:export-products"/>
-        </exec>
+    <target name="product-search-export-products" description="Exports all products for searching via elasticsearch." hidden="true">
+        <echo level="warning" message="This phing target is deprecated since Shopsys Framework 9.0.0, use 'elasticsearch-export' instead."/>
+        <phingcall target="elasticsearch-export">
+            <property name="elasticsearch.index" value="product"/>
+        </phingcall>
     </target>
 
-    <target name="product-search-migrate-structure" description="Creates new structure, reindexes it from old one, deletes old structure and add alias to new structure.">
-        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
-            <arg value="${path.bin-console}"/>
-            <arg value="shopsys:product-search:migrate-if-necessary"/>
-        </exec>
+    <target name="product-search-migrate-structure" description="Creates new structure, reindexes it from old one, deletes old structure and add alias to new structure." hidden="true">
```

Comment:
As all `product-search-` targets are now deprecated, it would be nice to change dependencies in other targets. Currently running tests, build, or so results in warnings in the console about the deprecated targets and the user have no way to fix it